acceptance/terrafarm/run.go

```diff
@@ -50,14 +50,14 @@ func (f *Farmer) run(cmd string, args ...string) (_ string, _ string, err error)
 		for scanO.Scan() {
 			line := scanO.Text() + "\n"
 			_, _ = outBuf.WriteString(line)
-			f.logf(line)
+			f.logf("%s", line)
```

Comment:
FYI: `line` here is from the terraform stdout (and stderr a few lines below). `logf` takes a format and list of args, so we really want it as an arg, otherwise formatting goes wonky when `%` is present.